inst/private/doctest_run_tests.m

```diff
@@ -40,25 +40,53 @@
   doctest_datastore('set_current_index', DOCTEST__i);
   DOCTEST__current_test = doctest_datastore('get_current_test');
 
+  DOCTEST__current_test.directive_issue = false;
```

Comment:
As far as I can see, we are not actually ever using this variable. Perhaps let's remove this for now until we replace this by a non-hacky implementation?